examples/pid/specification.yml

```
@@ -0,0 +1,42 @@
+user_claims:
+  iss: https://pid_issuer.memberstate.eu
+  iat: 1541493724
+  type: PersonIdentificationData
+  binding:
+  - !sd
+    type: CryptographicBinding
+    cnf:
+      jwk:
+        kty: EC
+        crv: P-256
+        x: TCAER19Zvu3OHF4j4W4vfSVoHIP1ILilDls7vCeGemc
+        y: ZxjiWWbZMQGHVWKVQ4hbSIirsVfuecCE6t4jT9F2HZQ
+  - !sd
+    type: BiometricBinding
+    template: "..."
+  !sd given_name: Erika
+  !sd family_name: Mustermann
+  !sd nationalities: [!sd "DE"]
+  !sd birth_family_name: Schmidt
+  !sd birthdate: '1973-01-01'
+  !sd place_of_birth: Regensburg
+  !sd address:
+    postal_code: '12345'
+    locality: Irgendwo
+    street_address: Sonnenstrasse 23
+    country_code: DE
+  !sd is_over_18: true
+  !sd is_over_21: true
+  !sd is_over_65: false
+
+holder_disclosed_claims:
+  binding:
+    - false
+    - true
+  nationalities:
+    - true
+  is_over_18: true
+  place_of_birth: true
+
+add_decoy_claims: true
```

```suggestion
add_decoy_claims: false
```

The nested decoy digests (e.g., https://drafts.oauth.net/oauth-selective-disclosure-jwt/sakurann-pid-ex/draft-ietf-oauth-selective-disclosure-jwt.html#appendix-A.5-9) make the disclosure content hard to follow. We might want to look at what the tooling does for nested decoys but in the meantime just not do decoys in this example. I think they only detract.